js/test/date/testdatefmt_bs_Cyrl_BA.js

```diff
@@ -540,7 +540,7 @@ module.exports.testdatefmt_bs_Cyrl_BA = {
             second: 0,
             millisecond: 0
         });
-        test.equal(fmt.format(date), "9");
+        test.equal(fmt.format(date), "с");
```

Comment:
If the date type is `m` and 'short` length, it should be the digit. no?

Reply:
In Bosnian, the "m" date component uses the stand-alone now. So, instead of the "m" date component mapping to the "M" format, it now maps to the "L" format, which uses letters instead of digits.

Reply:
little confused. What if a user wants to display with a number in that case?

Reply:
Then they would not use a date component specifier. The `date: "m"` means "give the format where the month name appears alone". If they wanted only the number, they would have to use the `template` property

Reply:
then what is https://github.com/iLib-js/iLib/blob/development/js/lib/DateFmt.js#L853 line for? 
In the root dateformats.json we have `n` (I think it should be `l`) others have  'l' (ru/dateformats.json)  I thought this character is for display standalone, and  character `m'  is to display digits.  no?